plugins/system/file_length_

```diff
@@ -15,6 +15,7 @@ Useful for things such as lists (white, black, user, ...).
     env.category DEFAULTS_TO_system
     env.title OPTIONAL_TITLE
     env.logarithmic 1
+    env.type DEFAULT_TO_GAUGE
```

Comment:
Make it `GAUGE` instead of `DEFAULT_TO_GAUGE`

Reply:
As I said, `DERIVE` looks like a better default. But I don't really use the plugin, so I don't really vote for any.